README.md

```diff
@@ -48,7 +48,7 @@ The official [helm chart](charts/kubernetes-external-secrets) can be used to cre
 
 ```bash
 $ helm repo add external-secrets https://external-secrets.github.io/kubernetes-external-secrets/
-$ helm install external-secrets/kubernetes-external-secrets
+$ helm install kes external-secrets/kubernetes-external-secrets
```

Comment:
It was somewhat intentionally left out, people should copy paste less 😄  
Think we had a suggestion before to add the generate-name flag 

what do you think about:
```suggestion
$ helm install [RELEASE_NAME] external-secrets/kubernetes-external-secrets
```

Please also update in [charts/kubernetes-external-secrets/README.md](https://github.com/external-secrets/kubernetes-external-secrets/blob/master/charts/kubernetes-external-secrets/README.md)

Reply:
Fair enough :)

Used your suggestion with `[RELEASE_NAME]` and updated both READMEs now.

Reply:
Thanks!